README.md

```diff
@@ -58,15 +58,15 @@ Here are a few other utility commands you may find useful:
 Sometimes you will want to use the latest version of the `web-design-standards` repo. Follow these steps to do so:
 
 1. Clone the latest version of the [`web-design-standards` repo](https://github.com/18F/web-design-standards/tree/staging).
-1. Run `npm install` to install the dependencies required for the package.
-1. Run `npm run build:package` to create the built version of the Standards.
+1. Run `npm install` to install the dependencies required for the package on the `web-design-standards` directory.
+1. Run `npm run build:package` to create the built version of the Standards on the `web-design-standards` directory.
 1. Run `npm link` in the _root level_ of the `web-design-standards` directory on your local machine.
 1. Run `npm link uswds` in the _root level_ of the `web-design-standards-docs` directory on your local machine.
-1. Run `npm run watch` in both projects to have changes automatically built and compiled on changes to any asset files.
-1. In a new terminal window, run `npm start` to start the Jekyll server locally.
+1. Run `npm run watch` in both projects to have changes automatically built and compiled on changes to any asset files (the `web-design-standards` and `web-design-standards-docs` directory).
```

Comment:
Thinking maybe: "Run `npm run watch` in both project directories to automatically build and compile changes when modifications are made to any asset files."